ubuntu-advantage

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh -e
+# shellcheck disable=SC2039
```

Comment:
This is to disable the warning about non-POSIX use of `local` (which is fine with dash)